.github/workflows/deploy_terraform.yml

```diff
@@ -96,25 +93,7 @@ jobs:
     steps:
       - name: Check Out Changes
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
-      - name: Connect to VPN and login to Azure
-        uses: ./.github/actions/vpn-azure
-        with:
-          env-name: ${{ needs.pre_job.outputs.env_name }}
-          tls-key: ${{ secrets.TLS_KEY }}
-          ca-cert: ${{ secrets.CA_CRT}}
-          user-crt: ${{ secrets.USER_CRT }}
-          user-key: ${{ secrets.USER_KEY }}
-          sp-creds: ${{ env.AZURE_CREDENTIALS }}
-          tf-auth: true
-      - name: Use specific version of Terraform
-        uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd
-        with:
-          terraform_version: 1.7.4
-          terraform_wrapper: false
       - name: Run Terraform
         run: |
           terraform init -input=false
-          terraform validate
-          terraform fmt -recursive
-          terraform plan -out ${{ needs.pre_job.outputs.env_name }}-tf.plan
-          terraform apply -input=false -no-color -lock-timeout=600s -auto-approve ${{ needs.pre_job.outputs.env_name }}-tf.plan
+          terraform apply -input=false -no-color -lock-timeout=600s -auto-approve
```

Comment:
We need tf plan here and corresponding tfplan file in apply:
`terraform plan -out ${{ needs.pre_job.outputs.env_name }}-tf.plan	
          terraform apply -input=false -no-color -lock-timeout=600s -auto-approve ${{ needs.pre_job.outputs.env_name }}-tf.plan`